lib/graph3d/Graph3d.js

```diff
@@ -956,52 +958,52 @@ Graph3d.prototype._dotSize = function() {
 
 
 /**
- * Get legend width 
```

Comment:
Just a comment for the next time: Please do not change code-style of other people. This messes up the "git blame".